projects/plugins/jetpack/extensions/plugins/publicize/store/selectors.js

```diff
@@ -22,36 +27,36 @@ const DEFAULT_TWEETSTORM_MESSAGE = '\n\n' + __( 'A thread ⬇️', 'jetpack' );
 /**
  * Returns the failed Publicize connections.
  *
- * @param {object} state - State object.
  * @returns {Array} List of connections.
  */
-export function getFailedConnections( state ) {
-	return state.connections.filter( connection => false === connection.test_success );
+export function getFailedConnections() {
```

Comment:
If these selectors don't accept `state` as the first parameter, is that an issue? As they are called on the store, `state` will be passed in.